modules/phase_field/src/materials/CrossTermBarrierFunctionMaterial.C

```diff
@@ -28,56 +28,75 @@ CrossTermBarrierFunctionMaterial::CrossTermBarrierFunctionMaterial(const InputPa
     _prop_g(declareProperty<Real>(_function_name)),
     _prop_dg(_num_eta),
     _prop_d2g(_num_eta)
+
 {
+  //loop counters
+  unsigned int i,j;
```

Comment:
Why move loop counters out of the local scope? It can only cause pain and suffering...